lib/ansible/modules/packaging/os/apt.py

```diff
@@ -38,6 +38,18 @@
       - Run the equivalent of C(apt-get update) before the operation. Can be run as part of the package installation or as a separate step.
     type: bool
     default: 'no'
+  update_cache_retries:
+    description:
+      - Amount of retries if the cache update fails. Also see I(update_cache_retry_max_delay).
+    type: int
+    default: 5
+    version_added: '2.9'
+  update_cache_retry_max_delay:
+    description:
+      - Use an exponential backoff delay for each retry (see I(update_cache_retries)) up to this max. delay in seconds.
```

Comment:
```suggestion
      - Use an exponential backoff delay for each retry (see I(update_cache_retries)) up to this max. delay in seconds.
```
The dot after “max” looks a bit strange